cogeo_mosaic/backends/dynamodb.py

```diff
@@ -244,3 +271,11 @@ def _table_exist(self) -> bool:
             return True
         except self.table.meta.client.exceptions.ResourceNotFoundException:
             return False
+
+    def _mosaic_exist(self) -> bool:
+        """Check if the mosaic already Exist in the Table."""
+        pass
+
+    def clean(self):
+        """clean MosaicID from dynamoDB Table."""
+        pass
```

Comment:
how to ?

Reply:
Unfortunately it's not possible to delete all items belonging to a single partition/hash key. I would recommend querying `_quadKeys` and doing a [batchWriteItem](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/dynamodb.html#DynamoDB.Client.batch_write_item) operation using the `DeleteRequest` Parameter

Reply:
check the latest commit ;-)